osbot_utils/utils/Misc.py

```diff
@@ -150,7 +150,7 @@ def is_number(value):
         pass
     return False
 
-def ignore_warning__unclosed_ssl():
+def ignore_warning_unclosed_ssl():
```

Comment:
ok if you want to make a change in an API like this, then please check where is it being used and also send a PR to that repo 

for example 👇 
![image](https://user-images.githubusercontent.com/57914593/107459831-e778b200-6b4e-11eb-9aba-0a33094ec3fe.png)

add a note to that PR that this will need to be merged first